LayoutTests/fast/events/cursors/mouse-cursor-resizer.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <style>
+        .target {
+            margin: 10px;
+            overflow: hidden;
+            width: 200px;
+            height: 100px;
+            resize: both;
+            text-align: right;
+            border: 12px solid silver;
+        }
+        #rtl-target {
+            direction: rtl;
+        }
+    </style>
+    <script src="../../../resources/js-test-pre.js"></script>
```

Comment:
Can you just use `js-test.js` (pre & post are old and shouldn't be used in newer tests).